composer.json

```diff
@@ -15,7 +15,7 @@
     ],
     "require": {
         "php": ">=5.3.6",
-        "symfony/console": "~2.1",
+        "symfony/console": "~2.4",
```

Comment:
Because the `Symfony\Component\Console\Output\BufferedOutput` is used and that was added in 2.4